app/workers/audio_callback_worker.rb

```diff
@@ -41,6 +42,9 @@ def perform(_sqs_msg, job)
 
     Shoryuken.logger.info("Updating #{job['type']}[#{audio.id}]: status => #{audio.status}")
     audio.save!
+    # announce the audio changes using the story as resource.
+    announce(:audio, :update, id: audio.story.id, resource: audio.story)
```

Comment:
Now that I think about it, it wouldn't actually create an infinite loop, since the lambda ignores any announcements where `status != UPLOADED`.

But ... I don't think we actually need/should announce the audio here.  I think we just want to `announce(:story` so that feeder gets notified that this file finished processing.